dlc_developer_config.toml

```diff
@@ -129,4 +129,4 @@ dlc-pr-tensorflow-2-graviton-inference = ""
 # EIA Inference
 dlc-pr-mxnet-eia-inference = ""
 dlc-pr-pytorch-eia-inference = ""
-dlc-pr-tensorflow-2-eia-inference = ""
+dlc-pr-tensorflow-2-eia-inference = ""
```

Comment:
can we please put the newline back at the bottom of this file? Code practice. When done, the toml file should not show up as a changed file.

Reply:
I will update this new line.

Reply:
Need the new line added back to the end of this file.